movai_core_shared/core/message_client.py

```diff
@@ -99,3 +99,6 @@ def foraward_request(self, request_msg: dict) -> None:
         """
         request = {"request": request_msg}
         self._zmq_client.send(request)
+        if request_msg["response_required"]:
+            return self._zmq_client.recieve()
```

Comment:
maybe we need to add timeout 
if the response will not come 
it will stuck the thread
and the function isn't async

Reply:
we set the timeout option on ZMQClient initialization look here: https://github.com/MOV-AI/movai-core-shared/blob/f90aeba5be69a3e8b6fc92e4c89e10f90cd02a4a/movai_core_shared/core/zmq_client.py#L37